src/pages/api/swagger.json

```diff
@@ -1,8 +1 @@
N","mr-IN","ms-MY","mt-MT","my-MM","nb-NO","ne-NP","nl-BE","nl-NL","nn-NO","or-IN","pa-IN","pl-PL","ps-AF","pt","pt-BR","pt-PT","ro-RO","ru","ru-RU","rw-RW","sc-IT","si-LK","sk-SK","sl-SI","sn-ZW","sq-AL","sr-RS","sv-SE","sw-KE","sy-SY","sz-PL","ta-IN","te-IN","tg-TJ","th-TH","tl-PH","tr","tr-TR","tt-RU","tz-MA","uk-UA","ur-PK","uz-UZ","vi-VN","zh","zh-CN","zh-HK","zh-MO","zh-TW","zz-TR"]}}],"responses":{"200":{"description":"OK"}}}},"/api/verification-codes":{"post":{"tags":["Verification Codes"],"parameters":[],"requestBody":{"required":true,"content":{"application/json":{"schema":{"oneOf":[{"type":"object","required":["email"],"properties":{"email":{"type":"string","format":"regex","pattern":"/^\\S+@\\S+\\.\\S+$/"}}},{"type":"object","required":["phone"],"properties":{"phone":{"type":"string","format":"regex","pattern":"/^\\d+$/"}}}]}}}},"responses":{"200":{"description":"OK"}}}},"/api/verification-codes/verify":{"post":{"tags":["Verification Codes"],"parameters":[],"requestBody":{"required":true,"content":{"application/json":{"schema":{"oneOf":[{"type":"object","required":["email","verificationCode"],"properties":{"email":{"type":"string","format":"regex","pattern":"/^\\S+@\\S+\\.\\S+$/"},"verificationCode":{"type":"string","minLength":1}}},{"type":"object","required":["phone","verificationCode"],"properties":{"phone":{"type":"string","format":"regex","pattern":"/^\\d+$/"},"verificationCode":{"type":"string","minLength":1}}}]}}}},"responses":{"200":{"description":"OK"}}}},"/api/phrase":{"get":{"tags":["Phrase"],"parameters":[{"name":"lng","in":"query","required":false,"schema":{"type":"string"}}],"responses":{"200":{"description":"OK"}}}},"/api/.well-known/sign-in-exp":{"get":{"tags":[".well-known"],"parameters":[],"responses":{"200":{"description":"OK"},"304":{"description":"No Modified"}}}},"/api/status":{"get":{"tags":["Status"],"parameters":[],"responses":{"204":{"description":"No Content"}}}},"/api/authn/hasura":{"get":{"tags":["Authn"],"parameters":[{"name":"resource","in":"query","required":true,"schema":{"type":"string","minLength":1}},{"name":"unauthorizedRole","in":"query","required":false,"schema":{"type":"string"}}],"responses":{"200":{"description":"OK"}}}},"/api/authn/saml/:connectorId":{"post":{"tags":["Authn"],"parameters":[{"name":"connectorId","in":"path","required":true,"schema":{"type":"string","minLength":1}}],"requestBody":{"required":true,"content":{"application/json":{"schema":{"type":"object","description":"arbitrary"}}}},"responses":{"200":{"description":"OK"}}}},"/api/swagger.json":{"get":{"tags":["Swagger.json"],"parameters":[],"responses":{"200":{"description":"OK"}}}}},"components":{"schemas":{"TranslationObject":{"type":"object","properties":{"[translationKey]":{"$ref":"#/components/schemas/Translation"}},"example":{"input":{"username":"Username","password":"Password"},"action":{"sign_in":"Sign In","continue":"Continue"}}},"Translation":{"oneOf":[{"type":"string"},{"$ref":"#/components/schemas/Translation"}]}}}}
```

Comment:
Is this a manual copy from @logto/core swagger.json? How should we keep it always up-to-date

Reply:
good call this is a mistake

Reply:
this file should not be checked in